src/robot/errors.py

```diff
@@ -183,9 +186,9 @@ def _get(self, error, attr):
 
 class ExecutionFailures(ExecutionFailed):
 
-    def __init__(self, errors, message=None):
+    def __init__(self, errors, message=None, attrs=None):
         super().__init__(message or self._format_message(errors),
-                         **self._get_attrs(errors))
+                         **(attrs or self._get_attrs(errors)))
```

Comment:
These changes shouldn't be needed anymore and should be reverted.

Reply:
Indeed, forgot to revert this change.